app/models/call.rb

```diff
@@ -1,8 +1,9 @@
-class Call
+ class Call < Active Record ::Base
```

Comment:
Two things here: 

* I'm not super sure we need to make our model class here descend from ActiveRecord but I might be wrong -- what do the docs say? 

* You nailed the inheritance syntax, but we'll want to change the spacing just a little bit since the name of the class is `ActiveRecord` -- if it turns out we do need to explicitly make these descend from ActiveRecord, let's do `class Call < ActiveRecord::Base`. 

Let me know if this doesn't make sense!